deploy/eck-stack/examples/apm-server/basic.yaml

```diff
@@ -0,0 +1,52 @@
+---
+eck-elasticsearch:
+  enabled: true
+
+  # Name of the Elasticsearch instance.
+  #
+  fullnameOverride: elasticsearch
+
+  nodeSets:
+  - name: default
+    count: 3
+    # Comment out when setting the vm.max_map_count via initContainer, as these are mutually exclusive.
+    # For production workloads, it is strongly recommended to increase the kernel setting vm.max_map_count to 262144
+    # and leave node.store.allow_mmap unset.
+    # ref: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-virtual-memory.html
+    #
+    config:
+      node.store.allow_mmap: false
+
+eck-kibana:
+  enabled: true
+
+  # Name of the Kibana instance.
+  #
+  fullnameOverride: kibana
+
+  spec:
+    config:
+      xpack.fleet.packages:
+      - name: apm
+        version: latest
```

Comment:
This is still not mentioned in the APM docs (I know we added it as part of https://github.com/elastic/cloud-on-k8s/pull/6063) but I wonder why they don't mention it.

Reply:
I'll craft something and get our docs updated, since it seems to be required.

Reply:
I was more thinking of the docs maintained by the APM team. But an update to our docs would be good too. Can be a separate PR from this one though.

Reply:
@pebrc Mind double-checking the wording added in [764038d](https://github.com/elastic/cloud-on-k8s/pull/7533/commits/764038da16dd25513fa6665a8c1dbd6ad72199c7)?

I will be added an issue concerning a larger update to the documentation concerning Kibana being a requirement, at least since 8.x.

Reply:
Taking a look now.